### PR TITLE
added Kubernetes examples & documentation

### LINKED ISSUE #396: Kubernetes example Manifest
I am using the ddns-Updater in my Kubernetes Home-Lab, i think it would be a good idea to provide some example Kubernetes manifests.

1. What's the feature?
Add some example K8s-manifests to deploy ddns-updater on Kubernetes. 

2. Extra information?
I think documentation and a Kustomize Base with Deployment and Service resources would be helpful for people who want to start with Kubernetes in their Home-Lab. Additional i would provide a Kustomize overlay with an Ingress resource, maybe with cert-manager support. 
Personally, i would reference this git with the manifest in my argocd-application, but this isn't relevant for the feature.  Just my personal use-case 😊.

Are you open for a PR of this kind?

### PR DESCRIPTION
**Pull request by 3deep5me**
fixes #396

### CODE REVIEW

File: k8s/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Advanced Install 
+
+### Overlays
+
+This folder also contains kustomize overlays which can extend the basic install.
+
+- [with-ingress](overlay/with-ingress/) - Basic **HTTP** Ingress ressource
+
+- [with-ingress-cert-manager](overlay/with-ingress-cert-manager/) - Basic **HTTPS** Ingress ressource which uses [cert-manager](https://github.com/cert-manager/cert-manager) to create certificates.
```
Comment (qdm12) [inline k8s/README.md:50]: Isn't the overlay directory missing? 🤔

Comment (3deep5me) [reply k8s/README.md]: you are right! I thought i did it. 
I try to do it this weekend till then i will put the PR in draft mode.

Comment (3deep5me) [reply k8s/README.md]: A little bit later but now it should fit.

**Comment by qdm12** [comment]
Merged thank you so much!!
Also sorry for the 6 months delay 😢 !

**Pull request merged**